end-to-end/artillery/queries/datasets.json

```diff
@@ -65,9 +65,9 @@
       "filter": { "term": { "entity_type.keyword": "Dataset" } },
       "aggs": {
         "donor.mapped_metadata.sex.keyword": {
-          "terms": { "field": "donor.mapped_metadata.sex.keyword", "size": 5 }
+          "terms": { "field": "donor..keyword", "size": 5 }
```

Comment:
I imagine this got changed while you were looking for example queries. These are queries used in load testing that should be kept as is. I would revert these changes.

Reply:
I did not touch this file. This is how it looks on my computer. I do not know why it looks different after I push the changes to github, because I did not do anything to this file. 
 
![Screen Shot 2021-07-07 at 1 06 19 PM](https://user-images.githubusercontent.com/85570580/124800901-30793a80-df24-11eb-8b7e-f25b32bf077e.png)

Reply:
@Roselkisma -- We understand that this is what your local version has, the problem is that this is a change from what is in master. (Red is master / green is yours.) It was an accident, it just needs to be undone. Not a big deal.